nirc_ehr/resources/scripts/nirc_triggers.js

```diff
@@ -70,4 +70,10 @@ exports.init = function (EHR) {
             removeTimeFromDate: false,
         });
     });
+
+    EHR.Server.TriggerManager.registerHandlerForQuery(EHR.Server.TriggerManager.Events.INIT, 'study', 'deaths', function(event, helper) {
```

Comment:
does this take precedence over the scriptOptions defined in deaths.js?

Reply:
Yes it does, the scriptOptions in deaths.js were getting overridden by the one in EHR, so ended up adding it here.

Reply:
Removed the one in death.js